runtime/battery-station/src/parameters.rs

```diff
@@ -98,13 +95,37 @@ parameter_types! {
     pub ContractsSchedule: pallet_contracts::Schedule<Runtime> = Default::default();
 
     // Court
-    /// Duration of a single court case.
-    pub const CourtCaseDuration: u64 = BLOCKS_PER_DAY;
+    /// (Slashable) Bond that is provided for overriding the last appeal.
+    /// This bond increases exponentially with the number of appeals.
+    /// Slashed in case the final outcome does match the appealed outcome for which the `AppealBond`
+    /// was deposited.
+    pub const AppealBond: Balance = 5 * BASE;
+    /// The blocks per year required to calculate the yearly inflation for court incentivisation.
+    pub const BlocksPerYear: BlockNumber = BLOCKS_PER_YEAR;
     /// Pallet identifier, mainly used for named balance reserves.
     pub const CourtPalletId: PalletId = COURT_PALLET_ID;
-    /// This value is multiplied by the current number of jurors to determine the stake
-    /// the juror has to pay.
-    pub const StakeWeight: u128 = 2 * BASE;
+    /// The time in which the jurors can cast their secret vote.
+    pub const CourtVotePeriod: BlockNumber = 3 * BLOCKS_PER_DAY;
+    /// The time in which the jurors should reveal their secret vote.
+    pub const CourtAggregationPeriod: BlockNumber = 3 * BLOCKS_PER_DAY;
+    /// The time in which a court case can get appealed.
+    pub const CourtAppealPeriod: BlockNumber = BLOCKS_PER_DAY;
+    /// The court lock identifier.
+    pub const CourtLockId: LockIdentifier = COURT_LOCK_ID;
+    /// The time in which the inflation is periodically issued.
+    pub const InflationPeriod: BlockNumber = 30 * BLOCKS_PER_DAY;
```

Comment:
I don't see any concerns about reducing the inflation period substantially, since the inflation emission is calculated per block by `issue_per_block`. People would get inflation faster in smaller chunks of money.

Reply:
Keep in mind, the overall computation per time is higher though..

Reply:
We have to test this on Battery Station and can't wait 1 month for results, can we reduce id to 7 or even 3 days on battery station?